esignet-integration-api/src/main/java/io/mosip/esignet/api/util/ErrorConstants.java

```diff
@@ -15,6 +15,7 @@ public class ErrorConstants {
     public static final String INVALID_AUTH_FACTOR_TYPE="invalid_auth_factor_type";
     public static final String INVALID_CHALLENGE="invalid_challenge";
     public static final String INVALID_CHALLENGE_FORMAT = "invalid_challenge_format";
+    public static final String INVALID_CHALLENGE_FORMAT_FOR_AUTH_FACTOR_TYPE = "invalid_challenge_format_for_auth_factor_type";
```

Comment:
This error code is present both in esignet-core and also in esignet-integration-api library?

Reply:
yes , for test cases we are fetching from esignet-core and I added beacause the other error coder are also present.